src/components/scenes/SendConfirmationScene.js

```diff
@@ -23,6 +23,7 @@ import { Slider } from '../../modules/UI/components/Slider/Slider.ui.js'
 import { type AuthType, getSpendInfoWithoutState } from '../../modules/UI/scenes/SendConfirmation/selectors'
 import { convertCurrencyFromExchangeRates } from '../../modules/UI/selectors.js'
 import { type GuiMakeSpendInfo, type SendConfirmationState } from '../../reducers/scenes/SendConfirmationReducer.js'
+import type { CurrencySetting } from '../../reducers/scenes/SettingsReducer'
```

Comment:
Why is this here?

Reply:
Originally it needed it when it was the main screen.

Reply:
It's used in its state props:

```
import type { CurrencySetting } from '../../reducers/scenes/SettingsReducer'
import { THEME } from '../../theme/variables/airbitz.js'
import type { GuiCurrencyInfo, GuiDenomination, GuiWallet } from '../../types/types.js'
import { scale } from '../../util/scaling.js'
import { convertNativeToDisplay, convertNativeToExchange, decimalOrZero, getDenomFromIsoCode } from '../../util/utils.js'
import { AddressTextWithBlockExplorerModal } from '../common/AddressTextWithBlockExplorerModal'
import { ExchangeRate } from '../common/ExchangeRate.js'
import { SceneWrapper } from '../common/SceneWrapper.js'
import { ButtonsModal } from '../modals/ButtonsModal'
import { Airship, showError } from '../services/AirshipInstance'

const DIVIDE_PRECISION = 18

export type SendConfirmationStateProps = {
  fiatCurrencyCode: string,
  currencyInfo: EdgeCurrencyInfo | null,
  currencySettings: CurrencySetting,
```